Modelica/package.mo

```diff
@@ -8487,6 +8487,7 @@ main version number is not changed.
 
 <table border=\"1\" cellspacing=\"0\" cellpadding=\"2\">
 <tr><td><a href=\"modelica://Modelica.UsersGuide.ReleaseNotes.Version_X_Y_0\">Version X.Y.0</a></td><td>Month D, 20YY</td></tr>
+<tr><td><a href=\"modelica://Modelica.UsersGuide.ReleaseNotes.Version_4_1_0\">Version 4.1.0</a></td><td>March 15, 2024</td></tr>
```

Comment:
```suggestion
<td>Month D, 20YY</td></tr>
<tr><td><a href=\"modelica://Modelica.UsersGuide.ReleaseNotes.Version_4_1_0\">Version 4.1.0</a></td><td>March 15, 2024</td></tr>
```
To me the version X_Y_0 was just a placeholder, and shouldn't be included in an actual release.
I checked and 4.0.0 didn't have that placeholder.

 A more radical solution would be to go back to the style of 3.2.2 and remove this entire table (assuming that tools will show the package contents anyway).

Reply:
I will commit with the placeholder removed.